src/components/venues/Jazzbar/EntranceExperience/EntranceExperience.tsx

```diff
@@ -206,99 +224,111 @@ const JazzbarEntranceExperience: React.FunctionComponent<PropsType> = ({
+                        return (
+                          <InformationCard
+                            title={venueEvent.name}
+                            key={venueEvent.id}
+                            className={`${!isNextVenueEvent ? "disabled" : ""}`}
+                          >
+                            <div className="date">
+                              {`${dayjs(startingDate).format("ha")}-${dayjs(
+                                endingDate
+                              ).format("ha")} ${dayjs(startingDate).format(
+                                "dddd MMMM Do"
+                              )}`}
                             </div>
-                          ) : (
-                            <div className="price-container">
-                              Individual tickets £{venueEvent.price / 100}
-                              <br />
-                              Group tickets £{venueEvent.collective_price / 100}
-                              {!user && (
-                                <div className="login-invitation">
-                                  Already have a ticket?{" "}
-                                  <span
-                                    className="link"
-                                    onClick={openAuthenticationModal}
-                                  >
-                                    Log in
-                                  </span>
-                                  .
-                                </div>
+                            <div className="event-description">
+                              {venueEvent.description}
+                              {venueEvent.descriptions?.map(
+                                (description, index) => (
+                                  <p key={index}>{description}</p>
+                                )
                               )}
                             </div>
-                          )}
+                            <div className="button-container">
+                              {hasUserBoughtTicket ? (
+                                <div>
+                                  <div>You have a ticket for this event</div>
+                                  <CountDown
+                                    startUtcSeconds={
+                                      venueEvent.start_utc_seconds
+                                    }
+                                  />
+                                </div>
+                              ) : (
+                                <div className="price-container">
+                                  Individual tickets £{venueEvent.price / 100}
+                                  <br />
+                                  Group tickets £
+                                  {venueEvent.collective_price / 100}
+                                  {!user && (
+                                    <div className="login-invitation">
+                                      Already have a ticket?{" "}
```

Comment:
remove the {" "} and use {\`Already have a ticket? \`}

Reply:
Annoyingly, the linter we are using (`npm run lint`) likes to do the {" "} so it can split JSX strings across lines. Whichever we go with it'd be great to immortalise it in a lint rule.

Reply:
ah that's annoying :( agree we should fix it at some point